api/api.go

```diff
@@ -0,0 +1,55 @@
+package api
+
+import (
+	"memberserver/api/auth"
+	"memberserver/config"
+	"memberserver/datastore"
+	"memberserver/resourcemanager"
+	"memberserver/resourcemanager/mqttserver"
+
+	"github.com/shaj13/go-guardian/v2/auth/strategies/jwt"
+	"github.com/shaj13/go-guardian/v2/auth/strategies/union"
+)
+
+// type Router interface {
```

Comment:
Can we add a comment here to explain why this is commented? If there is no reason then would it be alright to remove it?

Reply:
removed.